extensions/amp-connatix-player/0.1/amp-connatix-player.js

```diff
@@ -271,6 +271,7 @@ export class AmpConnatixPlayer extends AMP.BaseElement {
     const urlParams = dict({
       'playerId': this.playerId_ || undefined,
       'mediaId': this.mediaId_ || undefined,
+      'url': this.getAmpDoc().getUrl()
```

Comment:
This is unstable for the same document (URL may come from publisher's origin **or** a cache).

Consider using the AMP source URL instead. That way you always get the publisher's URL.

```suggestion
      'url': Services.documentInfoForDoc(this.element).sourceUrl,
```

Reply:
@alanorozco fixed